terraso_backend/apps/soil_id/models/project_soil_settings.py

```diff
@@ -20,6 +20,32 @@
 from apps.soil_id.models.depth_interval import BaseDepthInterval
 
 
+class DepthIntervalPreset(models.TextChoices):
+    LANDPKS = "LANDPKS"
+    NRCS = "NRCS"
+    NONE = "NONE"
+    CUSTOM = "CUSTOM"
+
+
+LandPKSPresets = [
+    dict(depth_interval_start=0, depth_interval_end=10, label="0-10 cm"),
+    dict(depth_interval_start=10, depth_interval_end=20, label="10-20 cm"),
+    dict(depth_interval_start=20, depth_interval_end=50, label="20-50 cm"),
+    dict(depth_interval_start=50, depth_interval_end=70, label="50-70 cm"),
+    dict(depth_interval_start=70, depth_interval_end=100, label="70-100 cm"),
+    dict(depth_interval_start=100, depth_interval_end=200, label="100-200 cm"),
```

Comment:
question: do the labels need to be localized?

Reply:
I think because the measurement abbreviations are pretty universal, we can probably get away with not localizing these.